carapace-server/src/main/java/org/carapaceproxy/core/Listeners.java

```diff
@@ -213,7 +213,7 @@ private void bootListener(NetworkListenerConfiguration config) throws Interrupte
                 .port(hostPort.getPort())
                 //.protocol(HttpProtocol.H2) // HTTP/2.0 setup
                 //.wiretap("logger", LogLevel.DEBUG, AdvancedByteBufFormat.TEXTUAL) // logging
-                .accessLog(true)
+                //.accessLog(true)
```

Comment:
Why disable it?

Reply:
We already got a custom accesslog to use in production, this is for debug purpose.